src/app/demos/image/index.html

```diff
@@ -0,0 +1,14 @@
+<stache pageTitle="Image" show showBreadcrumbs="false" showEditButton="true" showTableOfContents="true">
+
+  <stache-page-anchor>Simple image</stache-page-anchor>
+  <div style="width: 400px;">
+    <stache-image imageSource="~/assets/hero-sample-1.jpg"></stache-image>
+  </div>
+
+
+  <stache-page-anchor>Image with a description</stache-page-anchor>
+  <div style="width: 400px;">
+    <stache-image imageSource="~/assets/hero-sample-1.jpg" description="Figure 1: A hero image with a description."></stache-image>
```

Comment:
Thanks for updating the Demo to include this!  Once we square away the updates, can we also get an example of How this looks with Really long and really short descriptions?